node/Dockerfile

```diff
@@ -21,4 +20,18 @@ RUN --mount=type=cache,target=/root/.cache --mount=type=cache,target=/go \
   go build -race -gcflags="all=-N -l" --ldflags '-extldflags "-Wl,--allow-multiple-definition"' -mod=readonly -o /guardiand github.com/certusone/wormhole/node && \
```

Comment:
We don't want to run with `-race` in production

Reply:
Do you have a preferred solution for doing this? I'm thinking a build arg like `NO_SANITIZER` that's set to false by default